### ISSUE TITLE
CrystalLLM/CrystaLLM name issue

### ISSUE THREAD

**Issue by rgraucrespo** [facebookresearch/crystal-text-llm#7]
Hi - congratulations on this nice work. I would like to raise the issue of the rather unfortunate choice of name (CrystalLLM) that you have used for the code repository and in social media for your model. Our large language model for crystals, which you cite in your paper, is called CrystaLLM. Our model has been available as pre-print (https://arxiv.org/abs/2307.04340, cited a few times already) and webapp (https://crystallm.com) since July 2023, its source is openly distributed (https://github.com/lantunes/CrystaLLM), and it is currently undergoing peer review for journal publication. It is not in the best interest of the community to have two models for very similar purposes named practically the same thing (many people can/do easily refer to all of these variants when referring to CrystaLLM).  Would it be possible to address this issue? Kind regards, Ricardo.

**Comment by ngruver** [comment]
Hi Ricardo, 

Thank you for reaching out and for your great work on using autoregressive language models for generating crystals. I apologize for the oversight on my part. We've submitted a request to change this repo's name to crystal-text-llm, and I've added a note in the readme to point readers looking for CrystaLLM to your paper. Hopefully these changes will help disambiguate the two papers and methods. 

Nate
